opencti-platform/opencti-graphql/src/modules/event/event.graphql

```diff
@@ -105,7 +105,7 @@ type Event implements BasicObject & StixCoreObject & StixDomainObject & StixObje
     aliases: [String]
     # Technical
     creators: [Creator!]
-    toStix: String
+    toStix(version: Version): String
```

Comment:
Are you sure this parameter is optional defined like this?
We need to be able to only use toStix without any argument to prevent breaking change

Reply:
yes it is. in graphQL if there is no `!`, parameter is optionnal. I tested for all toStix.